data/matsim/.gitignore

```
@@ -1,3 +1,3 @@
 output
 network.xml.gz
-plans.xml.gz
+plans.xml.gz
```

If this is where we're putting the CSV files we generate for MATSim, we probably should ignore those. They would also change constantly
```suggestion
plans.xml.gz
*.csv

```